src/components/ChapterInfoPanel/spec.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { chapter } from '../../../tests/fixtures/chapters';
+import { chapterInfo } from '../../../tests/fixtures/chapterInfos';
+import ChapterInfoPanel from './index';
+
+let wrapper;
+let props;
+
+describe('<ChapterInfoPanel />', () => {
+  beforeEach(() => {
+    props = {
+      setOption: jest.fn(),
+      isShowingSurahInfo: true,
+      chapter,
+      chapterInfo
+    };
+
+    wrapper = shallow(<ChapterInfoPanel {...props} />);
+  });
+
+  it('should render', () => {
```

Comment:
This will cover `isShowingSurahInfo: true` and we should add spec for false case. "should not render it". But not blocker :)